LBHackney-IT/lbh-fss-portal-frontend
# Feature/update user

**Description**

- Added tests for EditUser component
- Added styling to EditUser component 
- Add responsive menu navigation bar
- Updated tests for Logout component 
- Updated tests for ProtectedLayout component 
- Updated functionality for updating user info

**Test instructions**

- run tests for Logout, ProtectedLayout and EditUser components and check they all pass 
- check responsive navbar works as expected 
- navigate to **/users**, click on a user and edit details - note: you should be able to update email, name or roles without having to enter a password, however if any text is entered into the password fields the password validation rules will take effect (minimum of 6 characters, at least 1 capital letter, at least one number). If update is success you will see success message. These same rules regarding password fields should apply in the same way on the **/account** page 
- navigate to **/users**, click on a user and-click on 'Delete account' you should see a responsive modal with 'Cancel' and 'Delete' options 
   - 'Cancel' will close the modal 
   - 'Delete' will redirect to **/users** page with a success message 

EditUser tests:
![image](https://user-images.githubusercontent.com/69238887/92101226-b85a5980-edd4-11ea-89ec-cfb2e80e7f05.png)

Logout tests:
![image](https://user-images.githubusercontent.com/69238887/92122638-9de3a880-edf3-11ea-89b4-9a066a535b69.png)

ProtectedLayout tests:
![image](https://user-images.githubusercontent.com/69238887/92137853-988f5980-ee05-11ea-8e28-3fda4f24cffd.png)

**Ticket**

https://app.clickup.com/t/9e5u94
